.gitignore

```diff
@@ -13,4 +15,5 @@ docs/_build
 .cache*
 .idea/
 integration-test/
-tests-env/
\ No newline at end of file
+tests-env/
+.pytest_cache/
```

Comment:
These changes should probably be in a separate PR as they are unrelated to import changes

Reply:
I have removed that by amending my commit.